rdf-star-cg-spec.html

```diff
@@ -558,22 +562,119 @@ <h2>Evaluation Semantics</h2>
     <section>
       <h2>Query Result Formats</h2>
 
-      <p>TODO: brief introduction paragraph (including a note that result of a CONSTRUCT query or a DESCRIBE query can be serialized using <a href="#turtle-star">Turtle*</a>)</p>
-
+      <p>In SPARQL, queries can take four forms: <em>SELECT</em>, <em>CONSTRUCT</em>, <em>DESCRIBE</em>, and <em>ASK</em>. [<a href="SPARQL11-QUERY#queryForms">SPARQL11-QUERY, Section 16</a>] The first of these returns its query solution as a set of variable bindings. The second and third both return an RDF graph, and the last returns a simple boolean value.
```

Comment:
This probably wants to be `<a data-cite="SPARQL11-QUERY#queryForms">SPARQL1.1 Query, Section 16</a> [[SPARQL11-QUERY]]` to use ReSpec citations properly.

Reply:
Thanks Greg, fixed.